### ISSUE TITLE
"Missing" application with install_zip_app

### ISSUE THREAD

**Issue by ttasovac** [bkuhlmann/mac_os#1]
## Expected Behavior

With the following setup:

```export UNICODECHECKER_APP_NAME=UnicodeChecker.app
export UNICODECHECKER_APP_URL="http://earthlingsoft.net/UnicodeChecker/UnicodeChecker.zip"
```

```
# UnicodeChecker
install_zip_app "$UNICODECHECKER_APP_URL" "$UNICODECHECKER_APP_NAME"
```

the application in question should install like all others. 

## Actual Behavior

Instead, when running the application script, I get this:

```
Downloading http://earthlingsoft.net/UnicodeChecker/UnicodeChecker.zip...
Preparing...
Installing: /Applications/UnicodeChecker.app...
cp: /tmp/downloads/UnicodeChecker.app: No such file or directory
Installed: UnicodeChecker.app.
 - Missing: UnicodeChecker.app
```

The zipped file is correctly downloaded to `/tmp/downloads` and unzipped, but, but it gets unzipped as a folder `UnicodeChecker 1.18` which contains `UnicodeChecker.app`. The script doesn't seem to be able to find it there. 

## Steps to Recreate

See above.  

P.S. Many thanks for this excellent collection of scripts.

**Comment by bkuhlmann** [comment]
Interesting find. Turns out the zip file builds the sub-directory structure dynamically upon unzipping. I've released a fix and if you upgrade to [v1.3.1](https://github.com/bkuhlmann/mac_os/blob/master/CHANGES.md#v131-2017-01-08) you'll be able to install successfully.

I'm glad you like these scripts and happy to hear they are of use to you. Thanks for reporting this bug!

BTW, that UnicodeChecker is a handy app, thanks -- a bit more verbose than [keycod.es](http://keycod.es) or [Keycode Info](http://keycode.info).
